app/samples/openrocket_sensors/data/flight.csv

```diff
@@ -0,0 +1,511 @@
+# Simulation 1 (Up to date)
```

Comment:
Still not a fan of csv in this codebase, but don't think theres much we can do about it now. Maybe in the future, we might store them in NSW? Idk just throwing ideas out

Reply:
Yea for this one where its specifically an example for this sample it feels like it should be right here. But yea for like next years irec prolly should go in NSW (or if we get CI/CD on the openrocket file can submodule that output :P)